requirements.in

```diff
@@ -0,0 +1,6 @@
+-e .[async,cli,pandas,sqlalchemy]
+pandas<0.21
```

Comment:
Most of these requirements are to satisfy Python 3.4 which is no longer supported.